.github/workflows/license-checker.yaml

```
@@ -0,0 +1,45 @@
+  schedule:
+    - cron: "0 3 * * *"
+
+jobs:
+  license-check-golang-backend:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Golang | Install Golang
+        uses: actions/setup-go@v4
```

actionlint (1.7.4)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)